### PR TITLE
Add MIT LICENSE

### PR DESCRIPTION
**Pull request by duminoreDev**
Before noone could use the code in their projects
Add MIT LICENCE for anyone to use our code

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Copyright (c) 2024 SHIZZLE
```
Comment (ellnix) [inline LICENSE]: SHIZZLE is unprofessional, how about "Highdashers Developer Group"?

**Review by ellnix — CHANGES_REQUESTED** [review]
Great idea!

File: LICENSE (hunk 1 above)
Comment (duminoreDev) [reply LICENSE]: Changed, check again :)

**Review by ellnix — APPROVED** [review]
Great idea, good job listening to me :D

**Pull request merged**